packages/sui-studio/src/components/demo/index.js

```diff
@@ -139,34 +174,73 @@ export default class Demo extends Component {
           <ContextButtons
             ctxt={ctxt || {}}
             selected={ctxtSelectedIndex}
-            onContextChange={this.handleContextChange} />
+            onContextChange={this.handleContextChange}
+          />
           <ThemesButtons
             themes={themes}
             selected={themeSelectedIndex}
-            onThemeChange={this.handleThemeChange} />
+            onThemeChange={this.handleThemeChange}
+          />
           <EventsButtons events={events || {}} store={store} domain={domain} />
         </div>
 
         <button className='sui-StudioDemo-codeButton' onClick={this.handleCode}>
-          {isCodeOpen ? iconClose : iconCode }
+          {isCodeOpen ? iconClose : iconCode}
+        </button>
+
+        <button
+          className='sui-StudioDemo-fullScreenButton'
+          onClick={this.handleFullScreen}
+        >
+          {isFullScreen ? iconFullScreenExit : iconFullScreen}
         </button>
 
-        {isCodeOpen && <CodeEditor
-          isOpen={isCodeOpen}
-          onChange={playground => { this.setState({playground}) }}
-          playground={playground}
-        />}
+        {isCodeOpen && (
+          <CodeEditor
+            isOpen={isCodeOpen}
+            onChange={playground => {
+              this.setState({ playground })
+            }}
+            playground={playground}
+          />
+        )}
 
         <Preview
           code={playground}
-          scope={{React, [`${cleanDisplayName(Enhance.displayName)}`]: Enhance, domain, ...nonDefaultExports}}
+          scope={{
+            React,
+            [`${cleanDisplayName(Enhance.displayName)}`]: Enhance,
+            domain,
+            ...nonDefaultExports
+          }}
         />
       </div>
     )
   }
 
   handleCode = () => {
-    this.setState({isCodeOpen: !this.state.isCodeOpen})
+    this.setState({ isCodeOpen: !this.state.isCodeOpen })
+  }
+
+  handleFullScreen = () => {
+    this.setState({ isFullScreen: !this.state.isFullScreen }, () => {
+      const { isFullScreen } = this.state
+      const aside = document.querySelector('.sui-Studio-sidebar')
```

Comment:
Prefer getElementsByClassname over querySelector when selectors are classnames for performance concerns.

Reply:
Oki